packages/react-native-reanimated/__typetests__/common/inlineStylesTest.tsx

```diff
@@ -44,7 +44,7 @@ function InlineStylesTest() {
   }
 
   function InlineStylesTest4() {
-    const sv = useSharedValue(true);
+    const sv = useSharedValue('0');
```

Comment:
They are related. The original test was supposed to check if passing a string here would be caught, since it's not an allowed type. However, in previous versions it wasn't caught in types of React Native and since `AnimatedStyle` types are derived from React Native, we didn't catch it either. Now this test could finally get updated.